src/runner/index.js

```diff
@@ -547,6 +548,8 @@ export default class Runner extends EventEmitter {
         if (!options)
```

Comment:
Actually, we should do a merge here too. E.g. you define some options like the dashboard URL in config/CLI, but want to use the globally saved token.

Reply:
We should do it as a separate task: https://github.com/DevExpress/testcafe/issues/7090